client/js/app.js

```diff
@@ -13,4 +13,15 @@ angular
       });
 
     $urlRouterProvider.otherwise('todo');
-  }]);
+  }])
+  .run([
+    '$rootScope', 'LoopBackAuth', 'User',
+    function($rootScope, LoopBackAuth, User) {
+      $rootScope.$on('$stateChangeStart', function (event, toState, toParams) {
+        //var currentUserId = LoopBackAuth.load('currentUserId');
```

Comment:
is this meant to be commented out?

Reply:
TBH I have no idea what is this about :(